src/types/onyx/PersonalDetails.ts

```diff
@@ -0,0 +1,45 @@
+type PersonalDetails = {
+    /** ID of the current user from their personal details */
+    accountID: number;
+
+    /** First name of the current user from their personal details */
+    firstName?: string;
+
+    /** Last name of the current user from their personal details */
+    lastName?: string;
+
+    /** Display name of the current user from their personal details */
+    displayName?: string;
+
+    /** Is current user validated */
+    validated?: boolean;
+
+    /** Phone number of the current user from their personal details   */
+    phoneNumber?: string;
+
+    /** Avatar URL of the current user from their personal details */
+    avatar?: string;
```

Comment:
This should also be required

```suggestion
    avatar: string;
```

Reply:
Adjusted, thanks 👌